src/pykiso/lib/robot_framework/communication_auxiliary.py

```diff
@@ -60,9 +60,19 @@ def send_message(
         logger.info(f"send message {raw_msg} using {aux_alias}")
         return state
 
```

Comment:
Somehow Im missing functions in this file ? 
where do you set unset self.com_aux.queueing_event ? 

will this break the robot variant ?